### ISSUE TITLE
Multiple IP addresses (especially IPv4/IPv6 for dual stack)

### ISSUE THREAD

**Issue by dougbtv** [k8snetworkplumbingwg/whereabouts#14]
Mostly opening for my own notes, but, with IPAM CNI plugins you can return multiple addresses, would we like to allow users to specify say -- two ranges, IPv4 & v6.

**Comment by ashish-billore** [comment]
Yes, actually it makes sense. Could you please suggest if it is planned to be implemented. Thanks!

**Comment by zhanggbj** [comment]
Hi @dougbtv,

As k8s is already working on dual-stack support, I'm wondering if there's any plan or expected date to support dual stack for whereabouts? Thanks!

I'm asking since we have some scenarios using multus-cni + whereabouts in a dual-stack environment.

**Comment by cgoncalves** [comment]
Implemented in https://github.com/k8snetworkplumbingwg/whereabouts/pull/250
